src/main/java/com/facebook/openwifirrm/ucentral/gw/models/UserInfo.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifirrm.ucentral.gw.models;
+
+import java.util.List;
+
+public class UserInfo {
+    public String id;
+    public String name;
+    public String description;
+    public String avatar;
+    public String email;
+    public boolean validated;
+    public String validationEmail;
+    public long validationDate;
+    public long created;
+    public String validationURI;
+    public long lastPasswordChange;
+    public long lastEmailCheck;
+    public long lastLogin;
+    public String currentLoginURI;
+    public String currentPassword;
+    public java.util.List<String> lastPasswords;
```

Comment:
You've already imported this, don't put the full package path here (`List` instead of `java.util.List`)